scripts/test-examples.js

```diff
@@ -1,21 +1,29 @@
 const path = require('path');
 const glob = require('glob');
 const execSync = require('child_process').execSync;
+const fs = require('fs');
 
-const examples = glob.sync(
-  path.join(
-    __dirname,
-    '..',
-    'examples',
-    '!(default-theme|e-commerce|media|tourism|hooks|hooks-ssr)'
-  )
-);
+const examples = glob
+  .sync(path.join(__dirname, '..', 'examples', '!(react-native*)'))
```

Comment:
before this PR the react native examples *were* tested, is this change expected?

Reply:
The tooling didn't make it easy. I'll try on another branch and I'll let you know.